### DISCUSSION TITLE
Anyway to detect/add a callback function to when a user opens a selection modal?

### DISCUSSION THREAD

**Discussion by bennetthermanoff** [azeezat/react-native-select#58] [Q&A] [answered]
I'm trying to action a fetch to only occur on modal opening. Is there anyway to dispatch this on open?

BTW: love this package!

**Comment by azeezat** [comment, marked as answer]
Hi @bennetthermanoff! I'm glad you are finding the library useful. Can you try this?

`modalProps={{ onShow: () => console.log('API call should happen now') }}`
